src/main/java/com/wultra/security/userdatastore/userclaims/UserClaimsController.java

```diff
@@ -72,6 +72,19 @@ public ErrorResponse handleNotFoundException(final NotFoundException e) {
         return new ErrorResponse("NOT_FOUND", e.getMessage());
     }
 
+    /**
+     * Exception handler for {@link EncryptionException}.
+     *
+     * @param e Exception.
+     * @return Response with error details.
+     */
+    @ExceptionHandler(EncryptionException.class)
+    @ResponseStatus(HttpStatus.CONFLICT)
```

Comment:
Normally we just use a `400` error to avoid too many `4xx` error types.

Reply:
I have no strong opinion but 409 is accepted by our guides https://github.com/wultra/development-internal/wiki/HTTP-Status-Codes My original idea was to distinguish client error and server error. This is most probably not fixable by the client. Should we return 5xx instead?

- [ ] decide HTTP status code